src/main/java/uk/gov/hmcts/reform/sscs/job/DataMigrationJob.java

```diff
@@ -39,7 +40,12 @@ public boolean readyToRun() {
     public void process() {
         String languageColumn = isRollback ? EXISTING_LANGUAGE_COLUMN : MAPPED_LANGUAGE_COLUMN;
         log.info("Processing Interpreter data {} job", isRollback ? "rollback" : "migration");
-        migrationService.process(languageColumn);
+        try {
+            migrationService.process(languageColumn);
+        } catch (IOException e) {
+            log.error("{} job failed to decode encodedDataString", isRollback ? "rollback" : "migration", e);
```

Comment:
can we please have a test to cover this ?

Reply:
there is one